src/blog/three-new-features-on-ganache-to-improve-your-developer-experience/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Three new Ganache features to improve your developer experience
+hide:
+  - navigation
+---
+
+**By [Kingsley Arinze](https://twitter.com/heydamali)**
+
+![The Merge and what it means for Truffle](./what-is-new-in-ganache.jpg)
+
+Since the release of Ganache v7.0 in January, we’ve mostly focused our attention on bug fixes and UX improvements. Recently, we decided to expand our focus to also include new features that will help improve the developer experience for our users.
+
+In this post, we’ll touch on 3 of those new features that we've added to Ganache since the Ganache v7.0 release.
+
+## Zero-config mainnet forking now available in the browser
+
+Ganache v7.0 made it possible to run Ganache in the browser. Simply add this script to your HTML:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/ganache@7/dist/web/ganache.min.js"></script>
```

Comment:
We do NOT recommend using `@7` as the version, as this will end up using the latest `7` version; which means behavior could potentially change from one page load to the next. Users should pin with a specific version (as you did below). In our README we note that `the {VERSION} in the above path needs to be replaced with a version number or tag that is listed in [npm](https://www.npmjs.com/package/ganache?activeTab=versions)`.